.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,40 @@
+---
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Bug
+description: Some feature works incorrectly
+assignees: ["ferretdb-bot"]
+labels: ["code/bug"]
+body:
```

Comment:
Don't we need an "OS version / environment" field? So people can report how they ran FerretDB (in Docker, on OS, in a cloud etc)

Reply:
#1160